### DISCUSSION TITLE
Show the level of the element

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#1888] [Q&A] [answered]
when i load the model,i want  show the high-quality of the model,
for example, when load the element of arc,and i see this element is polygon,when i zoom to  (or enlarge) the element of arc,and  show the arc (after the tile load)
so when init load the imodel, i can set  show the level of the element ?

**Comment by pmconne** [comment, marked as answer]
The display system draws 3d tiles, not elements. The level of detail is based on the size of the tile in screen pixels which is based on zoom level. By default level of detail is calculated such that, assuming all tiles have finished loading, the geometric error will be approximately 1 pixel.
You cannot configure level of detail for specific elements.
You can configure the display system to select higher level of detail tiles than it would by default, by setting [TileAdmin.Props.defaultTileSizeModifier](https://github.com/imodeljs/imodeljs/blob/1ae61b8ae29853b93fc9933324be26c7f8f3c83c/core/frontend/src/tile/TileAdmin.ts#L882) to a value greater than 1 when starting up IModelApp. The potential tradeoffs to doing so are mentioned in the linked code.
